Formula/alertmanager.rb

```diff
@@ -0,0 +1,91 @@
+class Alertmanager < Formula
+  desc "Prometheus Alertmanager"
+  homepage "https://prometheus.io/docs/alerting/alertmanager/"
+  url "https://github.com/prometheus/alertmanager/archive/v0.20.0.tar.gz"
+  sha256 "4789ef95b09ba86a66a2923c3535d1bfe30a566390770784c52052c7c83ee1bc"
+
+  bottle do
+    cellar :any_skip_relocation
+    rebuild 1
+    sha256 "3e2f6180a0594933ac5eeb1f5574d9f07a45479b556702a426697ffa507328fd" => :catalina
+    sha256 "7a0549df192cd2b4c23a7322ff2d8096eb19774027f028cbeef5363fb5718862" => :mojave
+    sha256 "a68c839c3b0d5d63e768ab069f72b7a4d1c32b385d4d448a3de444b5044e95c3" => :high_sierra
+  end
```

Comment:
No bottle block. (Where do those checksums come from, anyway?)

Reply:
First thing first... My module is modify from my previous PR prometheus.rb
So I really dunno what this block means.
Will remove it then.